test/e2e/upgrade/upgrade.go

```diff
@@ -54,31 +51,24 @@ func NoTests() []upgrades.Test {
 func AllTests() []upgrades.Test {
 	return []upgrades.Test{
 		&adminack.UpgradeTest{},
-		controlplane.NewKubeAvailableWithNewConnectionsTest(),
```

Comment:
Any impact on removing those tests? I guess just rebuilding history for them?

Reply:
I don't think so, it's effectively a test rename which is something that happens fairly often. Should be ok.